### ISSUE TITLE
Motorboat does not seem to exist [1.16.3-3.1.0]

### ISSUE THREAD

**Issue by GekidoLukas** [Flaxbeard/ImmersivePetroleum#194]
So I tried to get a motorboat and I just could not find the item.
Also, I tried to summon it using the following command and the entity turns out to be completely invisible

`/summon immersivepetroleum:speedboat`

**Comment by mindforger** [comment]
Not implemented yet, please use the search first, this has been answered quite a few times already .. Make sure you check closed issues too

**Comment by GekidoLukas** [comment]
ah okay, sorry ^^"

**Comment by TwistedGate** [comment]
Yeah.. imma just leave this one open. Instead of closing it aswell, to keep people from making more dupes

**Comment by TwistedGate** [comment]
Gonna be back next release 729fac37554eb90282237ba7ede5acbeeac1c23b (1.16.5)
